docker/prod/Dockerfile

```diff
@@ -5,6 +5,8 @@ LABEL maintainer="operations@openproject.com"
 ARG BUNDLER_VERSION="2.5.13"
 ARG NODE_VERSION="20.9.0"
 ARG BIM_SUPPORT=true
+ARG USE_JEMALLOC=false
```

Comment:
I would suggest renaming this to something like `JEMALLOC_SUPPORT` which defaults to true, whilst keeping `USE_JEMALLOC` with `false` as the default.

This way we build the images with JEMALLOC support by default without forcing anyone to use it.
But if they do want to, they can simply enable it without having to build their own image.

One could also add the build arg in the docker workflow which builds the images and set it true there.
But I would find it less of a hassle and potentially less confusing if it's just supported (but not used) by default.

Reply:
So in that case we even don't need JEMALLOCK_SUPPORT and build images with that lib for all platforms.

Reply:
You're right. Just drop the JEMALLOC_SUPPORT var then and simply always install it, but only use it when USE_JEMALLOC is true.